src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Colors.values()[new Random().nextInt(Colors.values().length)].toString();
```

Comment:
Instead of using `toString()` to get the string representation of the enum constants, use the `name()` method. This is a more reliable way to get the string representation of an enum constant as per the checklist guidelines.